src/components/Navbar.tsx

```diff
@@ -1,61 +1,74 @@
-                <div className='justify-flex-end mt-1 mr-10'>
-                    <NavLink
-                        exact
-                        to={routes.explorerIntro.path}
-                        className='ml-10 hover:text-blue-800'
-                        activeClassName='text-blue-600'>
-                        Explorer
-                    </NavLink>
-                    <NavLink
-                        exact
-                        to={routes.toolkit.path}
-                        className='ml-10 hover:text-blue-800'
-                        activeClassName='text-blue-600'>
-                        Toolkit
-                    </NavLink>
-                    <NavLink
-                        exact
-                        to={routes.about.path}
-                        className='ml-10 hover:text-blue-800'
-                        activeClassName='text-blue-600'>
-                        About
-                    </NavLink>
-                    <NavLink
-                        exact
-                        to={routes.team.path}
-                        className='ml-10 hover:text-blue-800'
-                        activeClassName='text-blue-600'>
-                        Team
-                    </NavLink>
-                    <ExternalLink
-                        href='https://www.biorxiv.org/content/10.1101/2020.08.07.241729v2'
-                        className='ml-6 bg-white font-mono font-normal border-2 border-gray-600 rounded-lg p-2 hover:text-blue-600 hover:border-blue-600'>
-                        Preprint
-                    </ExternalLink>
-                    <ExternalLink
-                        href='https://github.com/ababaian/serratus/wiki'
-                        className='ml-6 bg-white font-mono font-normal border-2 border-gray-600 rounded-lg p-2 hover:text-blue-600 hover:border-blue-600'>
-                        Wiki
-                    </ExternalLink>
+        <>
+            <div className='min-h-full z-10'>
+                <div className='flex lg:hidden justify-between items-center'>
+                    <div className='w-20 h-8 m-3'>
+                        <NavLink exact to={routes.home.path}>
+                            <img src='/logo.png' alt='logo'></img>
+                        </NavLink>
+                    </div>
+                    <Hamburger
+                        onToggle={() => setHamburgerPressed(!hamburgerButtonPressed)}></Hamburger>
+                </div>
+                <div
+                    className={`${
+                        hamburgerButtonPressed ? 'flex flex-col' : 'hidden'
+                    } lg:flex lg:flex-row w-screen  bg-gray-50 sm:p-4 lg:justify-between items-center gap-y-4 z-10 border-b-2  border-gray-200 font-montserrat font-medium`}>
+                    <div className='w-20 h-8 flex align-middle'>
+                        <NavLink exact to={routes.home.path}>
+                            <img src='/logo.png' alt='logo'></img>
```

Comment:
+1 to @ababaian's comment in Slack, this logo is redundant

Reply:
fixed it